versioned_docs/version-6.x.x/getting-started.md

```diff
@@ -91,15 +93,13 @@ const sequelize = new Sequelize("sqlite::memory:");
 
 To experiment with the other dialects, which are harder to setup locally, you can use the [Sequelize SSCCE](https://github.com/papb/sequelize-sscce) GitHub repository, which allows you to run code on all supported dialects directly from GitHub, for free, without any setup!
 
-## New databases versus existing databases
+## New versus existing databases
 
-If you are starting a project from scratch, and your database is still empty, Sequelize can be used from the beginning in order to automate the creation of every table in your database.
-
-Also, if you want to use Sequelize to connect to a database that is already filled with tables and data, that works as well! Sequelize has got you covered in both cases.
+If the database of your new project is still empty, use Sequelize from the beginning to automate the creation of every table in your database. Sequelize can also connect to a database that is already filled with tables and data. You will learn more in the concepts section and other parts of the documentation.
 
 ## Logging
 
-By default, Sequelize will log to console every SQL query it performs. The `options.logging` option can be used to customize this behavior, by defining the function that gets executed every time Sequelize would log something. The default value is `console.log` and when using that only the first log parameter of log function call is displayed. For example, for query logging the first parameter is the raw query and the second (hidden by default) is the Sequelize object.
+By default, Sequelize will log every SQL query to console. To customize this behavior, use the `options.logging` option (defaults to `console.log`) and define the function that gets executed every time Sequelize would log something. By default, only the first log parameter of log function call is displayed. For example, for query logging the first parameter is the raw query and the second is the Sequelize object.
```

Comment:
I know you did not write that part but I don't really understand what the following means:

> By default, only the first log parameter of log function call is displayed. For example, for query logging the first parameter is the raw query and the second is the Sequelize object.